.flake8

```diff
@@ -5,6 +5,11 @@ max-line-length = 79
 max-complexity = 18
 select = B,C,E,F,W,T4,B9,Q0,N8,VNE
 exclude =
-    **migrations
-    venv
-    tests
\ No newline at end of file
+    .git,
+    __pycache__,
+    build,
+    dist,
+    migrations,
+    env,
+    node_modules,
+    some_other_directory/
```

Comment:
Leave only those which related to your project
Add blank line in the end of file